hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/PrefetchExecutor.java

```diff
@@ -134,4 +139,25 @@ private PrefetchExecutor() {
   static ScheduledExecutorService getExecutorPool() {
     return prefetchExecutorPool;
   }
+
+  public static int getPrefetchDelay() {
+    return prefetchDelayMillis;
+  }
+
+  @Override
+  public void onConfigurationChange(Configuration conf) {
```

Comment:
nit: please add info logging that config has changed.

Reply:
Corrected